### DISCUSSION TITLE
Check for errors

### DISCUSSION THREAD

**Discussion by mariannevanharten** [fisharebest/webtrees#4608] [Ideas]
When checking tree for errors and no errors are found, I prefer to see a message like "No errors found". When webtrees don't show a messages it seems the system is still looking for errors, isn't it?

Best regards,
Marianne

**Comment by fisharebest** [comment]
I see such a message.  What do you see?

<img width="714" alt="Screenshot 2022-11-22 at 11 22 04" src="https://user-images.githubusercontent.com/2306088/203301974-3cd5b706-cf8b-455d-943f-a7275a898994.png">

**Comment by mariannevanharten** [comment]
I guess you only see this message when the tree initially has no errors.
I hide some types of errors. When I hide all  types, there are no errors.
The box "No errors have been found" doesn't came up in my tree at this moment.
When hiding all types, I want to see a message, a little bit changed: "No **other** errors have been found"
